indico/modules/search/client/js/components/results/Contribution.jsx

```diff
@@ -37,7 +37,7 @@ export default function Contribution({
         <a href={url}>{title}</a>
       </List.Header>
       <List.Description styleName="description">
-        <Highlight text={description} highlight={highlight.description} />
+        {description && <Highlight text={description} highlight={highlight.description} />}
```

Comment:
I guess we always get a description as well as a highlight?

Reply:
What do you think about returning `null` in `Highlight` if there's neither text nor a highlight?

Reply:
I was undecided... it's the balance between responsibility and verboseness. I think it made more sense to have the parent decide whether or not the child should render. On the other hand, the highlight shouldn't be empty. So if you agree, it's fine for me.

Reply:
I don't have a strong opinion either, but a slight preference on having the check in the component since that's what's rendering the `<div styleName="summary">` causing the margin anyway.